cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -146,6 +147,27 @@ users:
 		t.Fatal(err)
 	}
 
+	// plugin config
+	pluginconfigFile := filepath.Join(tmpDir, "plugin.yaml")
+	if err := ioutil.WriteFile(pluginconfigFile, []byte(fmt.Sprintf(`
+apiVersion: kubescheduler.config.k8s.io/v1alpha1
+kind: KubeSchedulerConfiguration
+clientConnection:
+  kubeconfig: "%s"
+plugins:
+  reserve:
+    enabled:
+    - name: foo
+    - name: bar
```

Comment:
Could you also add disabled plugins here for testing purposes?

Reply:
Updated. PTAL.
Thanks!!